### PR TITLE
question show styling

### LINKED ISSUE #41: question show styling
Adjust font colors, card background colors.

### PR DESCRIPTION
**Pull request by tjgragg**
Closes #41

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -23,3 +23,7 @@ nav {
 .brand-logo img {
  height: 60px;
 }
+
+.card .card-action a {
+  color: #039be5;
```
Comment (kurtisnelson) [inline app/assets/stylesheets/application.scss]: Hm, do we want this globally?

Comment (tjgragg) [inline app/assets/stylesheets/application.scss]: That link is changing the "edit" and "update" links on the question from that bright orange to match the links on the Classroom's website. So personally, I would think so. Unless there are some situations that I am not aware of that that line would effect.

Comment (kurtisnelson) [inline app/assets/stylesheets/application.scss]: This changes all card actions everywhere. Another option is to do a color override https://github.com/mkhairi/materialize-sass#a-sass

Comment (tjgragg) [inline app/assets/stylesheets/application.scss]: I think I'll need you to explain how to utilize this file...

Comment (kurtisnelson) [inline app/assets/stylesheets/application.scss]: The idea is that you can set all the colors that materialize uses up front via variables.

**Comment by tjgragg** [comment]
Ready to merge, except for travis. I think.

**Pull request merged**